samples/text/04-02-video.md

```diff
@@ -6,10 +6,10 @@ title: "Video"
 
 The Electric Book template includes an include tag for easily inserting videos from YouTube and Vimeo. [See the docs for more]({{ site.canonical-url }}/docs/editing/video.html).
 
-In this example, we have a YouTube video.
+In this example, we have a YouTube video, set to start 10 seconds in.
```

Comment:
Your example below looks like it starts 14 seconds in?

Reply:
Thanks! Fixed in 35dd013.